### ISSUE TITLE
Reformat discord messages

### ISSUE THREAD

**Issue by dudethatbe** [datafruits/datafruits#1076]
The messages displayed from Discord are a little too long and could be simplified to take up less space. @mcfiredrill suggested using the font-awesome icon and I think that sufficiently explains the origin of the message, but that may just be presumptuous of me...
Here's a mock-up of what I'm thinking 
![chat messages in datafruits displaying the differences in the proposed format against the existing format.](https://user-images.githubusercontent.com/2829501/222931543-3bd03961-dec3-43c9-a532-c564e6a40a97.png)

The elixir app seems to be treating all messages the same, despite where they came from https://github.com/datafruits/fruitbot/blob/e0a58d54c8f370b1db88af7b23171eefa146d42c/lib/fruitbot/worker.ex#L43

It would be nice to take this further and replace the "coach" username with the discord icon and the username of who actually sent the msg. I also think it would make more sense if the avatar and the discord icon were side-by-side. This is where I'm left wondering what everyone else thinks. Any ideas?

**Comment by mcfiredrill** [comment]
Sounds good to me, I think the format could be like
[avatar] [username] [discord icon] [message]

**Comment by dudethatbe** [comment]
So here's what I got so far. Since coach is sleeping and I'm not, I faked it by sending the exact phrase he uses for the relayed message. I'm going to add an additional check that if verifies the message was sent from coach. The icon is clickable too. It links to the discord invite URL.

![reformatted chat message that shows the selected username and a discord icon](https://user-images.githubusercontent.com/2829501/222948953-898ad0be-ab15-40d6-8eac-f148fa009206.png)
